src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery.Test/ScenarioTests/A2A/AsrA2ATests.ps1

```diff
@@ -0,0 +1,146 @@
+
+    $v = New-AzureRmRecoveryServicesAsrAzureToAzureDiskReplicationConfig -managed -LogStorageAccountId $logStorageAccountId `
+         -DiskId "diskId" -RecoveryResourceGroupId  $RecoveryResourceGroupId -RecoveryReplicaDiskAccountType  $RecoveryReplicaDiskAccountType `
+         -RecoveryTargetDiskAccountType $RecoveryTargetDiskAccountType
+
+     Assert-True { $v.LogStorageAccountId -eq $LogStorageAccountId }
+     Assert-True { $v.DiskId -eq $DiskId  }
+     Assert-True { $v.RecoveryResourceGroupId -eq $RecoveryResourceGroupId }
+}
+
+<#
+.SYNOPSIS 
+    Test GetAsrFabric new parametersets
+#>
+function Test-NewAsrFabric {
+    $seed = 35;
+        $vaultRgLocation = getVaultRgLocation
+        $vaultName = getVaultName
+        $vaultLocation = getVaultLocation
+        $vaultRg = getVaultRg
+        $primaryLocation = getPrimaryLocation
+        $recoveryLocation = getRecoveryLocation
+        $primaryFabricName = getPrimaryFabric
+        $recoveryFabricName = getRecoveryFabric
+        
+        New-AzureRmResourceGroup -name $vaultRg -location $vaultRgLocation -force
+        [Microsoft.Azure.Test.TestUtilities]::Wait(20 * 1000)
+    # vault Creation
+        New-azureRmRecoveryServicesVault -ResourceGroupName $vaultRg -Name $vaultName -Location $vaultLocation
+        [Microsoft.Azure.Test.TestUtilities]::Wait(20 * 1000)
+        $Vault = Get-AzureRMRecoveryServicesVault -ResourceGroupName $vaultRg -Name $vaultName
+        Set-ASRVaultContext -Vault $Vault
+    # fabric Creation    
+        ### AzureToAzure New paramset 
+        $fabJob=  New-AzureRmRecoveryServicesAsrFabric -Azure -Name $primaryFabricName -Location $primaryLocation
+        WaitForJobCompletion -JobId $fabJob.Name
+        $fab = Get-AzureRmRecoveryServicesAsrFabric -Name $primaryFabricName
+        Assert-true { $fab.name -eq $primaryFabricName }
+        Assert-AreEqual $fab.FabricSpecificDetails.Location $primaryLocation
+}
+
+
+function Test-NewContainer{
+
+    $seed = 33;
+    $primaryPolicyName = getPrimaryPolicy
+        $recoveryPolicyName = getRecoveryPolicy
+        
+        $primaryContainerMappingName = getPrimaryContainerMapping
+        $recoveryContainerMappingName = getRecoveryContainerMapping
+        
+        $primaryContainerName = getPrimaryContainer
+        $recoveryContainerName = getRecoveryContainer
+        $vaultRgLocation = getVaultRgLocation
+        $vaultName = getVaultName
+        $vaultLocation = getVaultLocation
+        $vaultRg = getVaultRg
+        $primaryLocation = getPrimaryLocation
+        $recoveryLocation = getRecoveryLocation
+        $primaryFabricName = getPrimaryFabric
+        $recoveryFabricName = getRecoveryFabric
+        $RecoveryReplicaDiskAccountType = "Premium_LRS"
+        $RecoveryTargetDiskAccountType = "Premium_LRS"
+
+        New-AzureRmResourceGroup -name $vaultRg -location $vaultRgLocation -force
+        [Microsoft.Azure.Test.TestUtilities]::Wait(20 * 1000)
```

Comment:
Should be using `[Microsoft.Rest.ClientRuntime.Azure.TestFramework.TestUtilities]::Wait`